docs/code-of-conduct.rst

```diff
@@ -0,0 +1,33 @@
+Code of Conduct
```

Comment:
Please see https://github.com/writethedocs/www/pull/115/commits/ef1be4587e35ad99a53575591c92ef8508ad1142 for comments